app/controllers/idv/review_controller.rb

```diff
@@ -26,11 +26,23 @@ def new
 
     def create
       resolution = start_idv_session
+      track_idv_event(resolution)
       process_resolution(resolution)
     end
 
     private
 
+    def track_idv_event(resolution)
+      result = {
+        success: resolution.success,
+        idv_attempts_exceeded: idv_attempter.exceeded?,
+        kbv: FeatureManagement.proofing_requires_kbv?,
+        new_phone_added: phone_confirmation_required?
```

Comment:
OK, so it looks like there are 2 separate IdV events we need to track. One is whether the initial resolution passed or not (from the Review Controller). The other is whether they completed IdV after this initial resolution, which may or may not involve a phone confirmation, and may or may not involve KBV.